config/locales/de.yml

```diff
@@ -599,10 +605,17 @@ de:
   registered_by: Registriert durch
   relatives: Familienangehörige im Haushalt
   remember_me: Angemeldet bleiben
+  reminder_mailer:
+    reminder_email:
+      please_confirm: 'Bitte bestätigen Sie die Stundenrapporte und das Feedback zum Einstatz %{client}.'
+      subject: Rapporte bestätigen
+  reminder_sent: 'Erinnerung wird zugestellt an %{email}'
```

Comment:
"versendet" instead of "zugestellt" sounds better IMHO

Reply:
Oh and just saw that the English version is in the past while this is in the future, maybe it should be "Erinnerung wurde versendet ..."?